app/controllers/api/v1/diaper_calculators_controller.rb

```diff
@@ -36,6 +36,14 @@ def diapers_service_handler(age)
       def childs_age
         params[:childs_age].to_i
       end
+
+      def product_price
+        diaper = Product.find_by title: 'diaper'
```

Comment:
logic to retrieve diaper_product should be in different method, because it's another logic, please keep in mind about single responsibility principe